ISHPermissionKit/ISHPermissionsViewController.m

```diff
@@ -205,6 +205,10 @@ - (void)permissionRequestViewController:(ISHPermissionRequestViewController *)vc
         [self transitionToPermissionCategoryAtIndex:nextIndex];
         [self setCurrentIndex:self.currentIndex + 1];
     }
+
+    if (error != nil && self.errorBlock) {
```

Comment:
`if (error & self.errorBlock)`? It's a little inconsistent to have an explicit and an implicit nil check side by side.